puppet/modules/slave/manifests/init.pp

```diff
@@ -114,6 +114,12 @@
       name => $osfamily ? {
         Debian => "libaugeas-dev",
         default => "augeas-devel"
+      };
+    "lib-qt":
+      ensure => present,
+      name => $osfamily ? {
+        Debian => "libqt4-dev",
+        default => "libqt4-devel"
```

Comment:
It's a mix of EL6 and Debian hosts.  I don't particularly want to have software installed from source on our servers, and it'd take a lot of effort to implement.

I don't think you can rely on that gem until we can rebuild the environment onto EL7.